IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcFeatureElementAddition/Documentation.md

```diff
@@ -4,7 +4,7 @@ The _IfcFeatureElementAddition_ is associated to its master element by virtue of
 
 The local placement for _IfcFeatureElementAddition_ is defined in its supertype _IfcProduct_. It is defined by the _IfcLocalPlacement_, which defines the local coordinate system that is referenced by all geometric representations. The local placement is always defined in relation to the local placement of the element to which the feature element is added:
 
-* The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point to the local placement of the same _IfcElement_, which is used in the _HasAdditionFeature.RelatingElement_ inverse attribute. 
+* The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point to the local placement of the _IfcElement_, which is used in the _ProjectsElements.RelatingElement_ inverse attribute. (the parent element of the feature).
```

Comment:
```suggestion
* The _PlacementRelTo_ attribute of _IfcObjectPlacement_ shall point to the object placement of the _IfcElement_, which is used in the _ProjectsElements.RelatingElement_ inverse attribute (the parent element of the feature).
```

Main improvement: the dot before the (...) is superfluous.

I'm unsure how do we currently handle these points - I've tried to make it better, but I'm open to discussion.

- all references to _PlacementRelTo_ in the documentation being an attribute of _IfcLocalPlacement_  shall be refactored to be of _IfcObjectPlacement_
- *local placement of an _IfcElement_* is somehow misguiding: it could as well be grid placement or linear placement

Reply:
@pjanck thanks for the corrections, 
> superfluous 
How snazzy! 

I did not look at the related locations of referenced properties hence the use of local as opposed to object. I will commit your suggestions!